app/controllers/wires_controller.rb

```diff
@@ -5,6 +5,7 @@ class WiresController < ApplicationController
 
   before_action :set_event, only: %i[new create]
   before_action :set_wire, only: %i[approve reject send_wire edit update]
+  before_action :enforce_sudo_mode, only: %i[create]
```

Comment:
technically some organizations are exempt from the $500 minimum so we should implement similar logic to https://github.com/hackclub/hcb/pull/11133 imo